src/components/Differential/ScatterPlotDiv.jsx

```diff
@@ -86,18 +87,32 @@ class ScatterPlotDiv extends Component {
       // user closes volcano plot, close options
       this.setState({ optionsOpen: false, usageOpen: false });
     }
+    if (
+      differentialResultsTableStreaming !==
+        prevProps.differentialResultsTableStreaming ||
+      // need this check for cached data from another test
+      differentialTest !== prevProps.differentialTest
+    ) {
+      this.getAxisLabels();
+    }
   }
 
   getMaxAndMin = (data, element) => {
     if (data.length > 0 && data[0][element] != null) {
       var values = [data[0][element], data[0][element]];
       for (var i = 1; i < data.length; i++) {
-        if (data[i] != null && data[i][element] != null) {
+        if (
+          data[i] != null &&
+          data[i][element] != null &&
+          !isNaN(data[i][element])
```

Comment:
some table data is "NA", and was passing this check, rather than forcing the min to be 0 (effectively disabling the transform checkbox)

Reply:
Turns out "NA" is fine.  So I'll adjust the PR description